src/main/java/co/cask/hydrator/plugin/common/MacroParser.java

```diff
@@ -0,0 +1,218 @@
+/*
```

Comment:
@bdmogal / @sreevatsanraman shouldn't this just be a class in CDAP and included? why does he need to include the file?